crates/file_store/src/store.rs

```diff
@@ -118,7 +126,7 @@ where
     /// **WARNING**: This method changes the write position in the underlying file. You should
     /// always iterate over all entries until `None` is returned if you want your next write to go
     /// at the end; otherwise, you will write over existing entries.
-    pub fn iter_changesets(&mut self) -> EntryIter<C> {
+    pub fn iter_changesets(&mut self) -> EntryIter<V> {
```

Comment:
I think it will be helpful to have two versions of this method. One that iterates over versioned changesets; `iter_versioned_changesets(&mut self) -> EntryIter<V>` and one that iterates over changesets; `iter_changesets(&mut self) -> EntryIter<C>`.

Reply:
Will do, thanks!